Source/SCM.SwissArmyKnife/Extensions/DictionaryExtensions.cs

```diff
@@ -51,5 +50,43 @@ public static class DictionaryExtensions
         {
             return dictionary.TryGetValue(key, out var value) ? value : valueProducer();
         }
+
+        /// <summary>
+        /// Returns a new dictionary after appling the provided function to each value from the given dictionary.
+        /// </summary>
+        /// <typeparam name="K">Key Type.</typeparam>
+        /// <typeparam name="O">Original Type.</typeparam>
+        /// <typeparam name="N">New Type.</typeparam>
+        /// <param name="dictionary">Source Dictionary.</param>
+        /// <param name="selector">Selector function.</param>
+        /// <example>
+        /// <code>
+        /// // Will return a new dictionary where value type is integer
+        /// var myDictionary = new Dictionary&lt;string, string&gt; {
```

Comment:
God I hate generics in XML comments. Such a mistake of Microsoft.

Reply:
yeah it is pain :)